home/panels.py

```diff
@@ -4,6 +4,14 @@
 
 class PageRatingPanel(Panel):
     class BoundPanel(Panel.BoundPanel):
+        template_name = "panels/page_rating_panel.html"
+
+        def get_context_data(self, parent_context=None):
+            context = super().get_context_data(parent_context)
+            context["page_value"] = self.instance.page_rating
+            context["revision_value"] = self.instance.latest_revision_rating
+            return context
+
         def render(self):
```

Comment:
Do we need to change or remove this function now? So that it uses the template instead? Otherwise this is template is duplicated in this function.

Reply:
@rudigiesler I think you are right, I went and looked, `render_as_object` is deprecated and the render shouldn't be necessary if there is a template, I have removed both :)